soc/riscv/riscv-privilege/telink_w91/Kconfig.n22

```diff
@@ -14,3 +14,13 @@ config TELINK_W91_FETCH_N22_BIN_REVISION
 	depends on TELINK_W91_FETCH_N22_BIN
 	help
 		This option sets N22 binary revision.
+
+DT_LABEL_Z_N22_PARTITION:= $(dt_nodelabel_path,n22_partition)
+
+config TELINK_W91_N22_PARTITION_ADDR
```

Comment:
N22 boot vector is already using this address?

Reply:
merge request is now to "telinlk_w91_matter_ota" branch. So, yes